docs/PUBLISHING.md

```diff
@@ -0,0 +1,92 @@
+# Overview
+We aim for the main branch of the repository to always be in a releasable state.
+
+Two types of artifacts can be published:
+1. Snapshot — An unstable release of the SDK for testing
```

Comment:
Shouldn't we call it something other than SDK here? Or is it some general naming convention for our dependency projects?